android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/CustomViewHolder.kt

```diff
@@ -47,6 +47,7 @@ sealed class CustomViewHolder(view: View) : RecyclerView.ViewHolder(view) {
         val addButton: ImageView,
         val allowedInPrivateBrowsingLabel: ImageView,
         val statusBlocklistedView: View,
+        val statusUnsignedView: View,
```

Comment:
Same as https://github.com/mozilla-mobile/firefox-android/pull/3840/files#r1343900573 could we just make `statusBlocklistedView` to accommodate both error messages?